### ISSUE TITLE
Checked Animate footprint does nothing

### ISSUE THREAD

**Issue by dch09** [rxhanson/Rectangle#1271]
- [x] This is a new bug that hasn't been documented in a closed issue or in the Readme.

macOS version: 14.0 Sonoma 
Rectangle version: v0.73(79)
Logs if applicable (In Rectangle menu, hold option, "View Logging..."):

**Comment by dch09** [comment]
Update: it does work from time to time. I'd say 1/15 resizes are actually animated

**Comment by thistlillo** [comment]
So, after 1 year, are there any news on this issue? I hit this page looking for the actual "role" of that option as nothing was changing when enabling/disabling "Animate footprint"....

**Comment by rxhanson** [comment]
Thanks for bringing this one back up. It never bubbled up to the top for me, mainly because I can't reproduce it. Would either of you mind showing a quick screen recording of the non-animated scenario? That might provide a little insight as to what's going on.

**Comment by aparolin** [comment]
Just sharing some thoughts here because maybe the feature is misunderstood? At least it was from my side :)
At first I thought this was going to animate the window itself when I moved it to the sides even with shortcuts like `control + option + <arrow keys>`. But after some testing, it seems this is supposed to animate only the transparent frame that predicts where the window will end up when dragging it to the sides of the screen. Is that correct?
If so, the feature seems to be working as expected on my MacBook Air M3 with macOS 26.6.1.

**Comment by rxhanson** [comment]
Good point, @aparolin. Indeed this setting is for animating the placeholder shadow window, not the actual window. I had looked at animating the actual window a long time ago and it was pretty terrible because of having to rely on the accessibility API to perform the move and resize separately - leading to some pretty wonky animations and lag. I'll take another look and see if I can come up with something better.
